cgaal-engine/Cargo.toml

```diff
@@ -1,12 +1,12 @@
 [package]
 name = "cgaal-engine"
-version = "1.0.1"
+version = "1.1.0"
```

Comment:
```suggestion
version = "1.0.1"
```
If anything, engine version should only bump PATCH number when the only change is updating emails in .toml

Reply:
But I'd rather that we don't increment.